src/scripts/main.js

```diff
@@ -1,5 +1,24 @@
 'use strict';
 
-// const pushNotification = (top, right, title, description, type) => {
-//   // write code here
-// };
+function pushNotification(top, right, title, description, type) {
+  const notification = document.createElement('div');
+
+  notification.innerHTML = `
+      <h2>${title}</h2>
+      <p>${description}</p>
+  `;
+
+  document.body.append(notification);
+  notification.classList.add('notification');
+  notification.classList.add(`${type}`);
```

Comment:
```suggestion
  notification.classList.add(type);
```
У тебя в type и так строка